site/scope-plugins.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Generating Custom Metrics with Plugins
+menu_order: 80
+---
+
+The following topics are discussed: 
```

Comment:
This could go in the developing section, as its primarily targeted at developers.